Dungeoneer/src/com/interrupt/dungeoneer/game/ModManager.java

```diff
@@ -421,8 +435,8 @@ public boolean checkIfModIsEnabled(String mod) {
         if(modsEnabled == null)
             return true;
 
-        Boolean enabled = modsEnabled.get(mod);
-        return enabled == null || enabled;
+            Boolean enabled = modsEnabled.get(mod);
+            return enabled == null || enabled;
```

Comment:
The tabs here are confusing.

Reply:
This can also be simplified to:
```java
return modsEnabled.getOrDefault(mod, false);
```

Reply:
Nope, that is breaking it. I tried something here as well, and that's when I broke the indentation here. Fixed.

Reply:
Fixed.